Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-MultipleFailedLogon_InShortSpan.yaml

```diff
@@ -1,6 +1,9 @@
 id: aef212b5-c770-42e1-9abf-bc513e4e749c
 name: Multiple Failed Logon on SQL Server in Short time Span
 description: |
+    This query looks multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function. 
```

Comment:
This is enough "This query looks multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function."